packages/react/src/components/TextField/TextField.styles.tsx

```diff
@@ -420,6 +421,7 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
       classNames.revealButton,
       'ms-Button',
       'ms-Button--icon',
+      getFocusStyle(theme),
```

Comment:
One more thing, I think the outline would be easier to see with a slight inset (this will also require updating snapshots again, sorry 😞)
```suggestion
      getFocusStyle(theme, { inset: 1 }),
```

Reply:
Ask and ye shall receive :) 

![image](https://user-images.githubusercontent.com/24663638/125709842-d195719c-578c-4395-8f66-a30eeb7fa617.png)